cli/operator/node.go

```diff
@@ -254,7 +256,15 @@ func setupDb(logger *zap.Logger, eth2Network beaconprotocol.Network) (basedb.IDb
 
 func setupOperatorStorage(logger *zap.Logger, db basedb.IDb) (operatorstorage.Storage, *registrystorage.OperatorData) {
 	nodeStorage := operatorstorage.NewNodeStorage(db)
-	operatorPubKey, err := nodeStorage.SetupPrivateKey(logger, cfg.OperatorPrivateKey, cfg.GenerateOperatorPrivateKey)
+	operatorPrivateKey := cfg.OperatorPrivateKey
+	if operatorPrivateKey == "" && !cfg.GenerateOperatorPrivateKey {
```

Comment:
Hi @SpontaneousOverthrow !
Thanks for the PR and contributing to SSV! 

Now to the review, don't you think it will be a better way to check if `OperatorPrivateKeyPath` is set?
Generally we need to decide on an order, let's say the config entry `OperatorPrivateKey` is highest priority, second is the file, and third and last is to generate a new key.

so to make things clearer - 
```
1. Check whether `cfg.OperatorPrivateKey` if it is use the key.
2. Check whether `cfg.OperatorPrivateKeyPath` is set, if it is use the key.
3. If both of the above are empty then we have no choice but to generate a new key. 
```

last point (doesn't have to be part of this PR) is maybe it'll be a good practice to save the key to a file even if we generated it, so it can be re-used in the future? or when for example migrating to a new server..

@SpontaneousOverthrow , @lior-blox your thoughts here will be appreciated.